experiment/measurer.py

```diff
@@ -588,15 +591,21 @@ def set_up_coverage_binary(benchmark):
     coverage_binaries_dir = build_utils.get_coverage_binaries_dir()
     benchmark_coverage_binary_dir = coverage_binaries_dir / benchmark
     if not os.path.exists(benchmark_coverage_binary_dir):
-        os.mkdir(benchmark_coverage_binary_dir)
+        try:
+            os.mkdir(benchmark_coverage_binary_dir)
+        except Exception as err:
```

Comment:
Are you sure this exception also needs to be caught?

Reply:
Yes. Based on the previous initial experiment #399.

Reply:
Basically, these 3 catching are related to the multiple initialization for benchmark coverage binaries preparation. And some race conditions happen locally.
I think the cloud running should also meet similar problem. The possible reason why it does not show up for cloud running, could be:
1. The different database implementation return different values in this function:
https://github.com/google/fuzzbench/blob/69feb9c327094033ced66ec2aea362c54bbbbb2a/experiment/measurer.py#L572-L582
Specifically, this line should only do one preparation for one benchmark. However, currently it will do the same amount as how many trials in total.
https://github.com/google/fuzzbench/blob/69feb9c327094033ced66ec2aea362c54bbbbb2a/experiment/measurer.py#L582
2. `gsutil` will handle them internally. However, I guess this is not the reason since some behavior only relate to `os` module.

Reply:
>1. The different database implementation return different values in this function:

[This may be possible](https://stackoverflow.com/questions/17223174/returning-distinct-rows-in-sqlalchemy-with-sqlite).
Can you try removing distinct and the catch and replacing it with `set()`?